### PR TITLE
Feature/change location

### PR DESCRIPTION
**Pull request by elizhahn**
## What does this PR do?
- The change location feature was implemented
- A method was added to change the user's location and return an updated list of trucks
- Upon submission the user will be taken to map and see an updated version of the map
- The service worker file was edited to prevent the service worker from updating the UI with old data

## How should it be tested?

- clear all storage and caches
- do a hard reload and sign in with user jackpot
- Change the location 2 times and make sure the map is being updated with new trucks, the console should be printing out a log of truck data
- go to the applications tab and check the offline option
- log out and refresh the page => you should see the page reload
- sign in with user jackpot
- You should be taken to a map with the previous truck data

## What do the future iterations hold?

- Additional testing, cleaning up the service worker file
- There are some console errors that may need to be addressed

## Linked Issues
- #58

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -26,6 +26,16 @@ export const fetchNewUser = (newUser) => {
 }
 
 export const fetchTrucks = (id) => {
+  console.log(id)
   return fetch(`https://warm-scrubland-95764.herokuapp.com/api/v1/trucks?id=${id}`)
   .then(response => checkForError(response))
+}
+
+export const updateUser = (updatedUser, id) => {
```
Comment (Shakikka) [inline src/apiCalls.js:34]: Nice work on implementing our first PATCH request.

File: src/components/App.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 html {
     height: 100%;
     // background: url('../assets/artboard_1.webp') no-repeat center center fixed;
-    background: url('../assets/37030.jpg') no-repeat center center fixed;
+    background: url('../assets/32050.jpg') no-repeat center center fixed;
```
Comment (Shakikka) [inline src/components/App.scss:4]: We'll have to take a group vote about this background image.

File: public/sw.js [hunk 3]
```diff
@@ -17,64 +20,277 @@ const urlsToCache = [
...
+        cacheNames.map(cacheName => {
+          if (cacheName !== CURRENT_CACHE) {
+            return caches.delete(cacheName);
           }
         })
       );
     })
   )
-})
+);
```
Comment (Shakikka) [inline public/sw.js:37]: I really hope this version works.

**Review by Shakikka — APPROVED** [review]

File: public/sw.js (hunk 3 above)
Comment (elizhahn) [reply public/sw.js:37]: on my knees praying

**Pull request merged**